src/input/hotkey.h

```diff
@@ -1,6 +1,9 @@
 #ifndef INPUT_HOTKEY_H
 #define INPUT_HOTKEY_H
 
+#include "SDL.h"
```

Comment:
Please, *no* SDL-related code/constants in anything other than the `platform` directory: if we decide to switch to something other than SDL, I want to keep the main codebase unchanged.